src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -1514,6 +1526,111 @@ describe_variable (InterpFrame *frame, MonoMethod *method, MonoMethodHeader *hea
 	return describe_value(type, addr, gpflags);
 }
 
+static gboolean
+decode_value (MonoType *t, guint8 *addr, const char* variableValue)
+{
+	char* endptr;
+	errno = 0;
+	switch (t->type) {
+	case MONO_TYPE_BOOLEAN:
+		if (!strcmp(variableValue, "True"))
+			*(guint8*)addr = 1;
+		else if (!strcmp(variableValue, "False"))
```

Comment:
Should we be doing case insensitive comparisons?

Reply:
As I could test these values always comes correctly from chrome. But if you think it's better to do case insensitive I can change, it's okay for me! :)

Reply:
no harm in doing that

Reply:
done.